app/src/util.cc

```diff
@@ -342,5 +342,15 @@ std::vector<std::string> SplitString(const std::string& s,
   return split_parts;
 }
 
+std::string CreateApiIdentifier(const char* api_id, void* object) {
```

Comment:
Suggestion: similar to what we do with FutureHandle, what about also having an ever-incrementing unsigned integer appended to the name? Or do we need CreateApiIdentifier to always return the same name for the same API/object?

Reply:
We don't, but I do think there is value in doing it based on address, if only for debugging purposes.  It is guaranteed to be unique as well, which is nice.

Reply:
It might be worth appending an incrementing number to the existing string, if only in case you delete a Firebase library and then it happens to get recreated in the same memory location.

Reply:
Ah, that's a good argument.  Added it.